src/fetchTokenBalance.ts

```diff
@@ -9,6 +9,6 @@ export async function fetchTokenBalance({
   tokenAddress: string;
   ownerAddress: string;
 }) {
-  const Token = new ethers.Contract(tokenAddress, ['function balanceOf(address account) view returns (uint256)'], provider);
+  const Token = new ethers.Contract(tokenAddress.toString(), ['function balanceOf(address account) view returns (uint256)'], provider);
```

Comment:
```suggestion
  const Token = new ethers.Contract(tokenAddress, ['function balanceOf(address account) view returns (uint256)'], provider);
```

here and in all other places this is unneeded change

Reply:
removed unnecessary toString() from address